README

```diff
@@ -4,15 +4,8 @@ Pluggable PayPal NVP (Name Value Pair) API implementation for Django based web a
 
 Installation
 ============
-    All you need to do is copying the "paypal" directory (or make a symbolink) under one of the directories in your PYTHON_PATH such as:
-
-    If you are using Python 2.5;
-    - /usr/local/lib/python2.5/site-packages/
-    - /usr/lib/python2.5/site-packages/
-
-    If you are using Python 2.6;
-    - /usr/local/lib/python2.6/dist-packages/
-    - /usr/lib/python2.6/dist-packages/
+    pip install -e git+django_paypal_driver_repository_url
+    All you need to do is copying the "paypal" directory (or make a symbolink) under one of the directories in your PYTHON_PATH.
```

Comment:
You've made it make no sense however. As there's no copying if you've pip installed it.